WordPress/src/test/java/org/wordpress/android/ui/jetpack/BackupStatusServiceTest.kt

```diff
@@ -46,7 +48,7 @@ class BackupDownloadStatusServiceTest {
     @Mock private lateinit var backupDownloadProgressChecker: BackupDownloadProgressChecker
     @Mock private lateinit var site: SiteModel
 
-    private lateinit var backupDownloadStatusService: BackupDownloadStatusService
+    private lateinit var mBackupStatusService: BackupStatusService
```

Comment:
Minor (🔍 ): I am not sure if we still want to keep using the Hungarian notation. If not, please consider removing the `m` suffix from this field. Wdyt?

Reply:
Thanks @ParaskP7 - I didn't realize AS did that. Yeah, I'll be changing that

Reply:
Yea, this is indeed strange, you got to love AS! 😅 Many thanks

Reply:
Pushed in c3d265e